crates/atuin/src/shell/atuin.zsh

```diff
@@ -27,6 +27,12 @@ fi
 export ATUIN_SESSION=$(atuin uuid)
 ATUIN_HISTORY_ID=""
 
+_atuin_update_title() {
+    if [[ "$TERM" =~ "xterm*" ]]; then
```

Comment:
Shouldn't this be 

```suggestion
    if [[ $TERM == xterm* ]]; then
```

? The regex match (`=~`) against `xterm*` matches `xter`, `xterm`, `xtermm`, `xtermmmm`, etc. Also, the regex matching is not a matching to the entire word but can match a substring of the supplied string.